charts/incubator/palworld/values.yaml

```diff
@@ -226,14 +226,23 @@ workload:
         main:
           probes:
             liveness:
-              type: tcp
-              port: "{{ .Values.service.rcon.ports.rcon.port }}"
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - pgrep -f "{{ .Values.persistence.serverfiles.targetSelector.main.main.mountPath }}/Pal/Binaries/Linux/PalServer-Linux-Test" >/dev/null || exit 1
             readiness:
-              type: tcp
-              port: "{{ .Values.service.rcon.ports.rcon.port }}"
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - pgrep -f "{{ .Values.persistence.serverfiles.targetSelector.main.main.mountPath }}/Pal/Binaries/Linux/PalServer-Linux-Test" >/dev/null || exit 1
```

Comment:
are you pgrep-ing a dir/file? 
This is not an indicator of a running proccess :D

Reply:
i ran it locally and echo that the process was running, this is near the same healthcheck that linuxgsm uses for their images